security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolationHelper.java

```diff
@@ -24,6 +24,14 @@ private static VoltageLevel getVoltageLevel(LimitViolation limitViolation, Netwo
         Objects.requireNonNull(limitViolation);
 
         Identifiable identifiable = network.getIdentifiable(limitViolation.getSubjectId());
+        if (limitViolation.getLimitType() == LimitViolationType.LOW_VOLTAGE_ANGLE || limitViolation.getLimitType() == LimitViolationType.HIGH_VOLTAGE_ANGLE) {
+            Optional<VoltageAngleLimit> limit = network.getVoltageAngleLimits().stream().filter(l -> l.getName().equals(limitViolation.getSubjectId())).findAny();
+            if (limit.isPresent()) {
+                return limit.get().getTerminalFrom().getVoltageLevel(); // why not?
+            } else {
+                // TODO
```

Comment:
?

Reply:
Not my code but I think the exceptiont throwing is missing.

Reply:
:-) You can change this part the way you want !

Reply:
Also `if (limit.isPresent()) {
                return limit.get()`is a bad use of options

Reply:
I think it depends on what you really want to do, but here orElseThrow should be enough.